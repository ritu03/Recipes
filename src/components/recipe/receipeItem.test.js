import React from 'react'
import { render, cleanup } from '@testing-library/react'

import RecipeItem from './recipeItem'


const Values = {
  imageList: {landscapewidedesktop1x: {url: 'abc.com'}},
  title: 'title123'
}

describe('RecipesItem snapshot', () => {
  afterEach(cleanup)

  test('renders properly', () => {
    const { container } = render(<RecipeItem values={Values} />)
    expect(container).toMatchSnapshot()
  })
})