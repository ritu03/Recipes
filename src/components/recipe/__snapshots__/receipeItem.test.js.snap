// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipesItem snapshot renders properly 1`] = `
<div>
  <div
    class="sc-bdnxRM djlCGH"
  >
    <img
      height="200"
      src="abc.com"
      width="250"
    />
    <div
      class="sc-gtsrHT kFRneN"
    >
      <p>
        title123
      </p>
    </div>
  </div>
</div>
`;
